modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -827,6 +827,14 @@ const register = (editor: Editor): void => {
     default: ''
   });
 
+  registerOption('force_hex_color', {
+    processor: (value) => {
+      const valid = Arr.contains([ 'always', 'rgb_only', 'off' ], value);
```

Comment:
Based on my other comment, we might be able to add a type to the array so we can catch future changes easier

Reply:
The force_hex_color processor now uses the ForceHexColor type